app/src/main/java/org/openobservatory/ooniprobe/utils/IntentCallback.java

```diff
@@ -0,0 +1,9 @@
+// TODO: since this is generally useful, move it into MK's libraries
```

Comment:
I haven't used the intent routing because I haven't understood the need. If you say they will be useful to run test in background from notification keep it, otherwise remove it